infrastructure/cloudconnect/README.md

```diff
@@ -0,0 +1,205 @@
+
+    ## BGP Example
+    $ aws ec2 describe-vpn-connections --filter Name=vpn-connection-id,Values=vpn-8e4251ef --query VpnConnections[0].CustomerGatewayConfiguration --output text | ./gcp-vpn-buildconf.py --network https://www.googleapis.com/compute/v1/projects/shared-vpc-host/global/networks/vpc >> config.bgp.yaml
+
+
+> Launch _gcp-vpn.jinja_ deployment
+
+**NOTE:** Run within the Shared VPC host project
+
+    ## Static Route Example
+    $ gcloud deployment-manager deployments create vpn-static-to-aws                 compute.v1.targetVpnGateway  COMPLETED  []
+    vpn-static-to-aws-rule-esp        compute.v1.forwardingRule    COMPLETED  []
+    vpn-static-to-aws-rule-udp4500    compute.v1.forwardingRule    COMPLETED  []
+    vpn-static-to-aws-rule-udp500     compute.v1.forwardingRule    COMPLETED  []
+    vpn-static-to-aws-tunnel1         compute.v1.vpnTunnel         COMPLETED  []
+    vpn-static-to-aws-tunnel1-route1  compute.v1.route             COMPLETED  []
+    vpn-static-to-aws-tunnel2         compute.v1.vpnTunnel         COMPLETED  []
+    vpn-static-to-aws-tunnel2-route1  compute.v1.route             COMPLETED  []
+
+    ## BGP Example
+    $ gcloud deployment-manager deployments create vpn-bgp-to-aws --config config.bgp.yaml
+    NAME                         TYPE                         STATE      ERRORS  INTENT
+    vpn-bgp-to-aws               compute.v1.targetVpnGateway  COMPLETED  []
+    vpn-bgp-to-aws-router        compute.v1.router            COMPLETED  []
+    vpn-bgp-to-aws-rule-esp      compute.v1.forwardingRule    COMPLETED  []
+    vpn-bgp-to-aws-rule-udp4500  compute.v1.forwardingRule    COMPLETED  []
+    vpn-bgp-to-aws-rule-udp500   compute.v1.forwardingRule    COMPLETED  []
+    vpn-bgp-to-aws-tunnel1       compute.v1.vpnTunnel         COMPLETED  []
+    vpn-bgp-to-aws-tunnel2       compute.v1.vpnTunnel         COMPLETED  []
+
+    ## BGP requires one execute a number of gcloud commands to create the tunnels and bgp peers. One can use the following bash command to run the preconfigured outputs in sequence.
+
+    ## Show Outputs
+    $ gcloud deployment-manager manifests describe --deployment vpn-bgp-to-aws | sed -n 's@^.*finalValue: @@p'
+
+    ## Execute Outputs
+    $ gcloud deployment-manager manifests describe --deployment vpn-bgp-to-aws | sed -n 's@^.*finalValue: @@p' | xargs -L1 bash
+    Updated [https://www.googleapis.com/compute/v1/projects/shared-vpc-host/regions/us-east1/routers/vpn-bgp-to-aws-router].
+    Updated [https://www.googleapis.com/compute/v1/projects/shared-vpc-host/regions/us-east1/routers/vpn-bgp-to-aws-router].
+    Updated [https://www.googleapis.com/compute/v1/projects/shared-vpc-host/regions/us-east1/routers/vpn-bgp-to-aws-router].
+    Updated [https://www.googleapis.com/compute/v1/projects/shared-vpc-host/regions/us-east1/routers/vpn-bgp-to-aws-router].
```

Comment:
Also the xargs command doesn't work for me as bash just exits.   I was able to run this instead:

gcloud deployment-manager manifests describe --deployment vpn-bgp-to-aws | sed -n 's@^.*finalValue: @@p'  | xargs -i@@ bash -c "@@"